accumulate_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"testing"
+	"time"
+)
+
+func Test(t *testing.T) {
+	zone := "appuio-cloudscale-ch-lpg"
+	tenant := "inity"
+	namespace := "testnamespace"
+
+	// get the correct date for a data set that was created yesterday
+	location, err := time.LoadLocation("Europe/Zurich")
+	if err != nil {
+		t.Errorf("could not load location Europe/Zurich")
+	}
+	now := time.Now().In(location)
+	date := time.Date(now.Year(), now.Month(), now.Day()-1, 0, 0, 0, 0, now.Location())
+
+	// build input data structure
+	var bucketMetricsInterval []cloudscale.BucketMetricsInterval
+	bucketMetricsInterval = append(bucketMetricsInterval, cloudscale.BucketMetricsInterval{
+		Start: date,
+		End:   date,
+		Usage: cloudscale.BucketMetricsIntervalUsage{
+			Requests:     5,
+			StorageBytes: 1000000,
+			SentBytes:    2000000,
+		},
+	})
```

Comment:
nitpick: could be simplified
```suggestion
	bucketMetricsInterval := []cloudscale.BucketMetricsInterval{
		{
			Start: date,
			End:   date,
			Usage: cloudscale.BucketMetricsIntervalUsage{
				Requests:     5,
				StorageBytes: 1000000,
				SentBytes:    2000000,
			},
		},
	}
```